test/adapter/json_api/links_test.rb

```diff
@@ -40,7 +40,7 @@ def test_toplevel_links
                   stuff: 'value'
                 }
               }
-            }).serializable_hash
+            }).serializable_hash(@options)
```

Comment:
:question: I don't think this should be here. Not defined anywhere and we haven't been passing anything interesting into the renderer options